examples/aosmith/schema/HeatSourceConfiguration.schema.yaml

```diff
@@ -0,0 +1,187 @@
+Schema:
+  Object Type: "Meta"
+  Title: "ASHRAE 205"
+  Description: "Base schema for ASHRAE 205 representations"
+  Version: "0.2.0"
+  References:
+    - "ASHRAE205"
```

Comment:
Because the HeatSourceTemplate is in the ASHRAE205 schema, it becomes a dependency for the CPP generator (the derived classes need to know where to find & scope the base class from. I posted a question on Slack about this issue...